manifests/custom_config.pp

```diff
@@ -115,9 +115,10 @@
       require     => Anchor['::apache::modules_set_up'],
     }
 
+    $remove_command = ['/bin/rm', shell_escape(join([$confdir, $_filename], '/'))]
```

Comment:
`shell_escape` shouldn't be needed when passing an array
```suggestion
    $remove_command = ['/bin/rm', join([$confdir, $_filename], '/')]
```

Reply:
My thought was that since I'm joining it and passing it through that way it may need to be escaped where it wouldn't if I was just passing it through.

Reply:
This does seem wrong now......
Have updated the code